src/components/card/ProductCard.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 
 const ProductCard = ({
+  title,
   image,
-  name,
-  description,
+  itemDescription = '',
   price,
   wishlisted = false,
```

Comment:
You could add some default values to these as those would act as a fallback when some product doesn't have any of such attributes in db. Without a fallback it would show empty space which is not a good UX.